plugins/language_gmi.lua

```diff
@@ -0,0 +1,22 @@
+-- mod-version:2 -- lite-xl 2.0
+local syntax = require "core.syntax"
+
+
+
+syntax.add {
+  name = "Gemtext",
+  files = { "%.gmi$" },
+  patterns = {
+    { pattern = { "```", "```" },           type = "string"   },
+    { pattern = "#.-",                      type = "keyword"  },
```

Comment:
If you want to highlight the whole line, you should use `#.*`.
Otherwise, if you only want to highlight the `#`, you should use `#+`

Reply:
Alright, I had just modified what was in the markdown plugin; I'll fix it here and fix it in the markdown plugin too.